openwisp_utils/qa.py

```diff
@@ -51,3 +53,166 @@ def check_migration_name():
         raise Exception("Migration %s %s in directory %s must "
                         "be renamed to something more descriptive."
                         % (file_, ', '.join(migrations), args.migration_path))
+
+
+def _commit_check_args():
+    """
+    Parse and return CLI arguements
+    """
+    parser = argparse.ArgumentParser(description="Ensures migration files "
+                                     "created have a descriptive name. If "
+                                     "default name pattern is found, "
+                                     "raise exception!")
+    parser.add_argument("--token",
+                        help="Github token for check commit")
+    parser.add_argument("--message",
+                        help="Commit message")
+    parser.add_argument("--repo",
+                        help="Repository target <username>/<repo>")
+    return parser.parse_args()
+
+
+def check_commit_message():
+    args = _commit_check_args()
+
+    commit = _get_commit()[0]
+
+    # Check args
+    # if not args.token:
+    #     raise Exception("CLI arguement `token` is required "
+    #                     "but not found")
+    if not args.repo:
+        raise Exception("CLI arguement `repo` is required "
+                        "but not found")
+    # Init var
+    # api = "https://api.github.com"
+
+    # If using token
+    # api_token = args.token
+
+    if args.message:
+        api_title = args.message.split("\n")[0]
+        if len(args.message.split("\n")) > 1:
+            api_message = args.message.split("\n")[1:]
+    else:
+        api_title = commit['title']
+
+        if commit['message'] != '':
+            api_message = commit['message'].split("\n")
```

Comment:
rename all variables that have the `api_` prefix so that they don't have it anymore